.mise.toml

```diff
@@ -1,3 +1,29 @@
+# whether to prompt to install plugins and runtimes if they're not already installed
+# missing_runtime_behavior = 'prompt' # other options: 'ignore', 'warn', 'prompt', 'autoinstall'
+
+# plugins can read the versions files used by other version managers (if enabled by the plugin)
+# for example, .nvmrc in the case of nodejs's nvm
+#legacy_version_file = true         # enabled by default (different than asdf)
+
+# configure `rtx install` to always keep the downloaded archive
+# always_keep_download = false        # deleted after install by default
+
+# configure how frequently (in minutes) to fetch updated plugin repository changes
+# this is updated whenever a new runtime is installed
+# (note: this isn't currently implemented but there are plans to add it: https://github.com/jdxcode/rtx/issues/128)
+#plugin_autoupdate_last_check_duration = 10080 # (one week) set to 0 to disable updates
+
+#verbose = false # see explanation under `RTX_VERBOSE`
+
 [tools]
-python = {version='3.11', virtualenv='.venv'}
-ruby = '3.2'
+python = ["3.11"]
+ruby = ["3.2.0"]
+node = ["20"]
+
+[settings]
+python_venv_auto_create = true
+experimental = true
```

Comment:
This change affects everything in the repo, right? I have no idea what this file does, but setting a global variable to `experimental` gives me the willies a bit.

Could we add a sentence at the top of this file that explains what it's for?

Also, what does `experimental = true` do for us?